### PR TITLE
Pass studio strings into validation to allow for localization 

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This allows Playlab levels to access the studio i18n strings in validation code.

There are only a handful of levels whose validation is locale-specific. As far as I could find, these levels use the "say" block. The strings for the say block are in `i18n/studio/en_us.json` already, so those can be reused. This isn't the cleanest solution, but is low-risk and will solve for the ~11 levels that need to be updated.

I'll update the levels in Levelbuilder after this PR gets there.

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/studio/studio.js [hunk 1]
```diff
@@ -6983,9 +6983,11 @@ var checkFinished = function() {
   var hasSuccessCondition =
     level.goal && level.goal.successCondition ? true : false;
   var achievedOptionalSuccessCondition =
-    !hasSuccessCondition || utils.valueOr(level.goal.successCondition(), true);
+    !hasSuccessCondition ||
+    utils.valueOr(level.goal.successCondition(studioMsg), true);
```
Comment (daynew) [inline apps/src/studio/studio.js:6987]: This seems like a simple solution to the problem you described, but I'm have trouble figuring out how all these pieces fit together. Could you help me understand?

`studioMsg` is all the translated strings, right? If so, why do we need to pass all the translated strings into this function? Why can't the implementation of `successCondition` currently access our translated strings?

I also don't understand where the `successConfition` function is defined which will make use of the translated screens.

Comment (bethanyaconnor) [reply apps/src/studio/studio.js:6987]: `successCondition` is an attribute of a level that contains a small javascript function that checks for "success".

For example, [this level](https://studio.code.org/s/coursec-2017/stage/13/puzzle/4) checks that a [certain sprite says a certain phrase](https://github.com/code-dot-org/code-dot-org/blob/d1dfb23a39e686ed4d37b013759363654b10e67d/dashboard/config/scripts/levels/courseC_PlayLab_events4.level#L9).
Screenshot 2020-06-30 at 10 20 54 AM

I couldn't get that function to access the translated strings on its own and I'm concerned about that being a sharp corner if we ever changed how files access the localized strings. We might forget to change these levels too.

Does that help?

Comment (bethanyaconnor) [reply apps/src/studio/studio.js:6987]: After this change, the strings can be accessed like:
Screenshot 2020-06-30 at 10 28 40 AM

**Comment by bethanyaconnor** [comment]
@daynew did I answer your question or do you have any other concerns?

**Review by daynew — APPROVED** [review]

**Pull request merged**